alicloud/resource_alicloud_ram_policy.go

```diff
@@ -122,12 +153,15 @@ func resourceAlicloudRamPolicyRead(d *schema.ResourceData, meta interface{}) err
 		return fmt.Errorf("GetPolicyVersion got an error: %#v", err)
 	}
 
-	d.Set("policy_name", policy.PolicyName)
-	d.Set("policy_type", policy.PolicyType)
+	policyDocument, err := ParsePolicyDocument(policyVersionResp.PolicyVersion.PolicyDocument)
+	if err != nil {
+		return err
+	}
+
+	d.Set("name", policy.PolicyName)
+	d.Set("type", policy.PolicyType)
 	d.Set("description", policy.Description)
-	d.Set("default_version", policy.DefaultVersion)
-	d.Set("attachment_count", policy.AttachmentCount)
-	d.Set("policy_document", policyVersionResp.PolicyVersion.PolicyDocument)
```

Comment:
Please retain default_version,  attachment_count  and policy_document in the state file, and modify 'policy_document' to 'document'.